From 98831f2e48eda6fcf067055214edc3f9c22f671d Mon Sep 17 00:00:00 2001
From: orhtej2 <2871798+orhtej2@users.noreply.github.com>
Date: Tue, 30 Dec 2025 14:11:30 +0100
Subject: [PATCH] Don't use zygote and GPU for export.

---
 exporter/src/app/browser.cljs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/exporter/src/app/browser.cljs b/exporter/src/app/browser.cljs
index 526ae77..463ffb4 100644
--- a/exporter/src/app/browser.cljs
+++ b/exporter/src/app/browser.cljs
@@ -100,7 +100,7 @@
 
 (def browser-pool-factory
   (letfn [(create []
-            (p/let [opts    #js {:args #js ["--allow-insecure-localhost" "--font-render-hinting=none"]}
+            (p/let [opts    #js {:args #js ["--allow-insecure-localhost" "--font-render-hinting=none" "--disable-gpu" "--no-zygote"]}
                     browser (.launch pw/chromium opts)
                     id      (swap! pool-browser-id inc)]
               (l/info :origin "factory" :action "create" :browser-id id)
-- 
2.39.5

